lusSTR/marker.py

```diff
@@ -915,10 +915,12 @@ def designation(self):
             if len(lus_sec) >= 2:
                 lus_allele = lus_sec[-1]
                 sec_allele = lus_sec[0]
+            elif len(lus_sec) > 2:
+                lus_allele = lus_sec[-1]
+                sec_allele = lus_sec[0]
             else:
                 lus_allele = 0
                 sec_allele = lus_sec[0]
```

Comment:
The logic of these conditionals is unclear to me. If the value is > 2, this will be handled by lines 916-917, and lines 919-920 will never be executed. Indeed, I commented out those lines and re-ran the test suite and it completed just fine. **UPDATE**: The contents of these blocks are identical anyway, so these lines should definitely be removed to avoid confusion or updated if they were intended to serve some other purpose.

Reply:
That was very strange and not quite sure how it changed, looking back at the commits. I changed it to the correct code and the tests are passing for me.